VmAgent.Core/VmConfiguration.cs

```diff
@@ -42,6 +42,11 @@ public class VmConfiguration
         // should use `PF_SHARED_CONTENT_FOLDER` instead (which points to wherever the folder is mounted in the container)
         public const string SharedContentFolderVmVariable = "PF_SHARED_CONTENT_FOLDER_VM";
 
+        // Used by the games to share user generated content (and other files that are downloaded once, used multiple times).
```

Comment:
can you add here the great explanation you put on the PR description? This will help the reader understand why we need two env variables.

Reply:
I expanded the comment (but it's still not as long as the novel i wrote for the PR description)